From f3756853aefa3d910f6d97b93e0656cadc9ea6b9 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Wed, 18 Mar 2020 12:30:44 -0700
Subject: [PATCH] configure.ac: Don't check for library

sqlite-modern-cpp is header-only, so don't abort if the build system has
trouble locating the sqlite3 library.

Fixes error:

| checking for sqlite3.h... yes
| checking for sqlite3_finalize in -lsqlite3... no
| configure: error: sqlite3 not found.
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 87b766b..1304672 100644
--- a/configure.ac
+++ b/configure.ac
@@ -87,7 +87,6 @@ fi
 
 sqlite=yes
 AC_CHECK_HEADERS(sqlite3.h, [], [sqlite=no])
-AC_CHECK_LIB(sqlite3, sqlite3_finalize, [], [sqlite=no])
 
 if test "$sqlite" == "no" 
 then
-- 
2.17.1

